app/presenters/content_change_presenter.rb

```diff
@@ -29,7 +30,12 @@ def call
   delegate :title, :description, :change_note, :footnote, to: :content_change
 
   def content_url
-    PublicUrlService.url_for(base_path: content_change.base_path)
+    utm_source = content_change.id
+    utm_medium = "email"
+    utm_campaign = "govuk-notifications"
+    utm_content = @frequency
+    base_path = "#{content_change.base_path}?utm_source=#{utm_source}&utm_medium=#{utm_medium}&utm_campaign=#{utm_campaign}&utm_content=#{utm_content}"
```

Comment:
At a fundamental level I'm wondering if we want the email_id available which can be used to determine more things (but is not fun to use).

This might not matter until it's decided it's wanted.

Reply:
That would make the analytics traceable to a particular person though, which may not be what we want

Reply:
Sure thing.